tools/toolchain/Dockerfile

```diff
@@ -13,5 +13,7 @@ RUN dnf -y update \
     && cp system-auth /etc/pam.d \
     && echo 'Defaults !requiretty' >> /etc/sudoers
 RUN mkdir -p /root/.m2/repository
+ADD --chmod=0555 https://dl.minio.io/server/minio/release/linux-amd64/minio /usr/local/bin/
```

Comment:
just wanted to make sure -- we only build the image for amd64 and we always test on amd64, right? i failed to find anything mentioning this where `dbuild` is referenced.

Reply:
Good question. Presumably no, it shouldn't be arch-specific

Reply:
Please add it to install-dependencies.sh and make it arch independent  (see node_exporter). We want it to be possible to run outside dbuild too.